Documentation/network/l2-announcements.rst

```diff
@@ -0,0 +1,332 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _l2_announcements:
+
+************************************
+L2 Announcements / L2 Aware LB
+************************************
+
+L2 Announcements is a feature which makes services visible and reachable on 
+the local area network. This feature is primarily intended for on-premises
+deployments within networks without BGP based routing such as office or 
+campus networks.
+
+When used, this feature will respond to ARP queries for ExternalIPs and/or 
+LoadBalancer IPs. These IPs are Virtual IPs (not installed on network 
+devices) on multiple nodes, so for each service one node at a time will respond
+to the ARP queries and respond with its MAC address. This node will perform 
+load balancing with the service load balancing feature, thus acting as a 
+north/south load balancer.
+
+The advantage of this feature over NodePort services is that each service can
+use a unique IP so multiple services can use the same port numbers. When using 
+NodePorts, it is up to the client to decide to which host to send traffic, and if a node
+goes down, the IP+Port combo becomes unusable. With L2 announcements the service
+VIP simply migrates to another node and will continue to work.
+
+The L2 Announcements feature and all the requirements can be enabled as follows:
+
+.. tabs::
+    .. group-tab:: Helm
+
+        .. parsed-literal::
+
+            $ helm upgrade cilium |CHART_RELEASE| \\
+               --namespace kube-system \\
+               --reuse-values \\
+               --set l2announcements.enabled=true \\
+               --set kubeProxyReplacement=strict \\
+               --set k8sServiceHost=${API_SERVER_IP} \\
+               --set k8sServicePort=${API_SERVER_PORT}
+               
+
+    .. group-tab:: ConfigMap
+
+        .. code-block:: yaml
+
+            enable-l2-announcements: true
+            kube-proxy-replacement: strict
+
+Prerequisites
+#############
+
+* Kube Proxy replacement mode must be enabled and set to ``strict`` mode, :ref:`kubeproxy-free` for details.
+
+* All devices on which L2 Aware LB will be announced should be enabled and included in the 
+  ``--devices`` flag or ``devices`` Helm option if explicitly set, see :ref:`NodePort Devices`.
+
+* The ``externalIPs.enabled=true`` Helm option should be set, if usage of externalIPs
+  is desired. Otherwise service load balancing for external IPs is disabled.
```

Comment:
Terminology: “should” or “must”?

(Same question for the use of “should” at other locations in the doc.)